site/styles/_footer.scss

```diff
@@ -52,16 +57,17 @@ footer {
 
 .footer-social-media-icons {
   display: grid;
-  gap: 24px;
-  grid-auto-columns: 25px;
+  gap: 16px;
+  grid-auto-columns: 32px;
   grid-auto-flow: column;
   grid-template-rows: 1fr;
   justify-content: center;
-  margin-bottom: 30px;
+
+  padding: 8px 16px;
```

Comment:
Core-template is supposed to have downward flowing margins... if you mix downward and upward margins (or padding), you risk things looking too squashed or spread out when elements render conditionally.  I would have a bottom margin on the ul nav, and another bottom margin on footer-social-media-icons, so each element is in control of how far away the next element is.

Reply:
I've fixed this. I needed to override the top and bottom padding method the nav links used to align them properly